contrib/colab/Distributing_Large_Embedding_tables.ipynb

```diff
@@ -282,27 +270,27 @@
         "    for i, _ in enumerate(self.parameters()):\n",
         "      if i != last_index:\n",
         "        yield _"
-      ],
-      "execution_count": null,
-      "outputs": []
+      ]
     },
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "5wahSapg6YAE",
-        "colab_type": "text"
+        "colab_type": "text",
+        "id": "5wahSapg6YAE"
       },
       "source": [
         "Let's now create the `Namespace`, which `fairseq` uses to define the task, dataset, model and more."
       ]
     },
     {
       "cell_type": "code",
+      "execution_count": null,
       "metadata": {
-        "id": "HCcPvjX36i_7",
+        "colab": {},
         "colab_type": "code",
-        "colab": {}
```

Comment:
just quick question, how did these changes happen?

Reply:
> Change is quite large, I thought this is just a wheel replacement?

Yea, they are all URL path changes.

Reply:
I only changed the pip url for torch_xla?

Reply:
And yes, if you look just above, you would see the same elements are tehre, e.g., `"colab": {}`. They are reordered.